cmd/ctr/commands/run/run_unix.go

```diff
@@ -172,7 +172,7 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 					oci.WithUserNamespace([]specs.LinuxIDMapping{uidMap}, []specs.LinuxIDMapping{gidMap}))
 				// use snapshotter opts or the remapped snapshot support to shift the filesystem
 				// currently the only snapshotter known to support the labels is fuse-overlayfs:
-				// https://github.com/AkihiroSuda/containerd-fuse-overlayfs
+				// https://github.com/containerd/fuse-overlayfs-snapshotter
```

Comment:
Should be a separate commit

Reply:
yes..
I will delete that in current commit.